src/Command/SynchronizeSubscribersCommand.php

```diff
@@ -2,17 +2,48 @@
 
 namespace Welp\MailchimpBundle\Command;
 
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use DrewM\MailChimp\MailChimp;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
-use Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException;
 use Welp\MailchimpBundle\Provider\ListProviderInterface;
+use Welp\MailchimpBundle\Subscriber\ListSynchronizer;
 
-class SynchronizeSubscribersCommand extends ContainerAwareCommand
+class SynchronizeSubscribersCommand extends Command
```

Comment:
Is this still compatible with SF2?

Reply:
I will check.

Reply:
Yes, the Command class exists in 2.8:
https://symfony.com/doc/2.8/console.html

The ContainerAwareCommand was for commands that needed the container.